Fails to add missing return values on Windows 10

goreturns doesn't add missing returns values and returns following with -p parameter

```
main.go:4:2: could not import errors (Config.Importer not installed)
main.go: typechecking failed (continuing without type info)
package main

import (
        "errors"
        "fmt"
)

func main() {
        fmt.Println("hello world")
}

func F() (int, error) {
        return errors.New("foo")
}
```
Running go 1.9 on Windows 10 64bit

--- a/returns/returns.go
+++ b/returns/returns.go
@@ -13,6 +13,7 @@ import (
 	"go/ast"
 	"go/build"
 	"go/format"
+	"go/importer"
 	"go/parser"
 	"go/printer"
 	"go/token"
@@ -120,6 +121,7 @@ func parseAndCheck(fset *token.FileSet, pkgDir, filename string, src []byte, opt
 			}
 			nerrs++
 		},
+		Importer: importer.Default(),
 	}
 
 	info := &types.Info{

